eng/pipelines/runtime-cet.yml

```diff
@@ -0,0 +1,148 @@
+
+trigger:
+  batch: true
+  branches:
+    include:
+    - release/*.*
+  paths:
+    include:
+    - '*'
+    - docs/manpages/*
+    exclude:
+    - /**/*.md
+    - eng/Version.Details.xml
+    - .github/*
+    - docs/*
+    - LICENSE.TXT
+    - PATENTS.TXT
+    - THIRD-PARTY-NOTICES.TXT
+
+pr:
```

Comment:
I believe this is what autotriggers the pipeline for PRs. I would remove the entire PR section for now, it will only become important once we decide to include CET testing in PR validation.